packages/storybook/stories/QuickStarts/snippets/QuickstartCompositeComplete.snippet.tsx

```diff
@@ -1,28 +1,69 @@
 import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
 import {
-  CallComposite,
   CallAdapter,
+  CallComposite,
   createAzureCommunicationCallAdapter,
-  ChatComposite,
   ChatAdapter,
+  ChatComposite,
   createAzureCommunicationChatAdapter,
-  fromFlatCommunicationIdentifier
+  fromFlatCommunicationIdentifier,
+  CompositeLocale,
+  CallCompositeOptions
 } from '@azure/communication-react';
+import { PartialTheme, Theme } from '@fluentui/react';
 import React, { useEffect, useMemo, useState } from 'react';
 
+export type CompositeProps = {
+  // Required fields.
+  userId: string; // '<Azure Communication Services Identifier>'
+  token: string; // '<Azure Communication Services Access Token>'
+  endpoint: string; // '<Azure Communication Services Resource Endpoint>'
+  displayName: string; // '<Display Name>'
+  groupId: string; // '<Developer generated GUID>'
+  threadId: string; // '<Get thread id from chat service>'
+
+  // Optional fields for customizing composites.
+  formFactor?: 'desktop' | 'mobile'; // '<Used to set the form factor for the calling composite>'
+  fluentTheme?: PartialTheme | Theme; // '<Theming for the composites using FluentUi themes>'
+  callInvitationURL?: string; // '<Invitation Url that will appear with a invite to call button>'
+  locale?: CompositeLocale; // '<Sets Locale of buttons in the composites>'
+  options?: CallCompositeOptions; // '<object that sets flags for visual elements of the composites>'
+  errorBar?: boolean; // '<Hides and shows the Errorbar in the chat composite>'
```

Comment:
we use options for CallCompositeOptions, but here we have expanded the ChatCompositeOptions
suggestion:
```
callCompositeOptions?: CallCompositeOptions; // '<set flags for visual elements of the composites>'
chatCompositeOptions?: ChatCompositeOptions; // '<set flags for visual elements of the composites>'
```